web-common/.gitignore

```diff
@@ -12,3 +12,4 @@
 **/.psc-package2nix/
 **/.spago/
 **/.spago2nix/
+node_modules
```

Comment:
Better not to merge this change to master. I think reinstating the web-common/package.json is probably a better solution to this problem, but in any case it's a separate problem, not for this PR.

Reply:
ok! Yeah I remember seeing something about this on another PR, I'll take it out.